packages/components/README.md

```diff
@@ -156,6 +156,31 @@ const SearchComponent = () => {
 
 In this example, the `Search` component receives the current search text and a handler function to update this text. The `loading` prop can be used to display a loading indicator, and the `debounce` prop controls the debounce delay.
 
+### List
```

Comment:
Insert `List` before `Markdown` to keep alphabetical order.

Reply:
updated